agent_build_refactored/docker_image_builders.py

```diff
@@ -755,7 +745,6 @@ class K8sRestartAgentOnMonitorsDeathAlpine(ContainerImageBuilderAlpine):
 ALPINE_K8S_IMAGE_BUILDERS = [
     K8sAlpine,
     K8sWithOpenMetricsAlpine,
```

Comment:
Since we are cleaning it up I guess we may just go ahead and delete K8sWithOpenMetricsAlpine + related code as well.